### DISCUSSION TITLE
Zero chia proofs/wins

### DISCUSSION THREAD

**Discussion by GHUser281** [Chia-Network/chia-blockchain#5057] [Support]
Hi,

I've started plotting/farming on 4/22/2021.
On 4/27 I had 213 plots and it was showing 9 days to win, in Chia UI.
I was continuing creating approximately 40-60plots a day, and farming on them, so all the time it was showing approximately 8-12 days to win. It's been like that 17 days and I still have 0 chia.
As of now (5/14), I have 920 K-32 plots.

I've enabled DEBUG in the logs. It's always "found 0 proofs", in under ~0.2sec, and 0 to up to 6 plots are eligible, I think ~1-2 plots are eligible, in average.
There were some Node Sync issues some days for some brief moments, but other than that I am out of ideas what's wrong.
Am I missing something else? 

Thank you!

**Comment by cccat6** [comment]
You need more luck.
It is possible you didn't get any reward. But keep farming.
If you are looking for a smother reward, wait for the pool out.

**Reply by GHUser281** [reply]
Yeah, unfortunately it's not a fair-random, but just a random, so the results might be different from what you expect or plan.
Hopefully this will be mitigated/solved with the pools. 
Thanks!

**Comment by lvcivs** [comment]
It's possible that you are just unlucky. 

One thing to check is whether your receive address for the farmer reward is set correctly. You can check that in the GUI.

**Reply by uwerui** [reply]
> It's possible that you are just unlucky.
> 
> One thing to check is whether your receive address for the farmer reward is set correctly. You can check that in the GUI.

Does the receiving address in the wallet have to be the same as the planting reward address in the farm?

**Reply by cccat6** [reply]
Not necessarily since you can get a lot of receiving addresses with your private key as you want. No need to be what shown in the GUI. But usually, the answer is YES.

**Reply by uwerui** [reply]
> Not necessarily since you can get a lot of receiving addresses with your private key as you want. No need to be what shown in the GUI. But usually, the answer is YES.

Is the farming reward address the same as the receiving address of the wallet?

**Reply by uwerui** [reply]
> Not necessarily since you can get a lot of receiving addresses with your private key as you want. No need to be what shown in the GUI. But usually, the answer is YES.

How to check the receiving address of private key?

**Reply by GHUser281** [reply]
Thanks for the reply.
According to the config xch_target_address under **farmer** and under **pool** are the same, and are equal to what's in UI.
I suspect even if that was incorrect, I would still see "found 1 proofs" in the log, right?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
